data/coffees.js

```diff
@@ -0,0 +1,90 @@
+module.exports = [
```

Comment:
nice work, do need to add a boolean value to your objects somewhere, left a comment on canvas as well.